zoom.sh

```diff
@@ -17,6 +17,8 @@ if [[ "$XDG_SESSION_TYPE" == "wayland" ]]; then
 
 		fi
        	fi
+        # Force XDG_CURRENT_DESKTOP to 'gnome' so that Zoom will not block portal screensharing
+        XDG_CURRENT_DESKTOP="gnome"
```

Comment:
This assumes `XDG_CURRENT_DESKTOP` is already set and exported, but it seems this might not be the case for some Wayland WMs/compositors like sway: https://news.ycombinator.com/item?id=31770930 (I'm not a Wayland user myself so can't verify this).

`export XDG_CURRENT_DESKTOP="gnome"` would likely make this work in those environments as well.

Reply:
`export` is only required if it needs to exist outside of the script shell, which it doesn't in this case.

Reply:
Doesn't it? This shell script doesn't use this variable, it's meant to be set for the `exec /app/extra/zoom/ZoomLauncher` invocation, and that only happens if it's `export`ed.